### ISSUE TITLE
Some tidy up required for Assistive Technology and Reading system entries

### ISSUE THREAD

**Issue by ways2read** [IDPF/epubtestweb#74]
I assume this is edits to the test entries. For example, JAWS is listed 6 different ways at the moment, so the filter cannot work correctly.

There is some improvement that can be made to the Reading systems list also, specifically to VitalSource Bookshelf.

This issue may be logged in the wrong place, so maybe should be moved.

**Comment by marisademeglio** [comment]
Yes this is a known data clean up issue that I've talked about on calls and on the mailing list. It stems from the way data was entered on the old site. 

The strategy we'll be using is to provide administrative tools for cleaning up data (the AT entries, like you mentioned; also, reading systems and result summaries). 

Filing this issue as a feature for our admin tools.
